### PR TITLE
Validate alertmanager URL

### LINKED ISSUE #1455: Validate command line parameters at start (especially --alertmanager.url)
Where we can, we should validate that command line parameters are correct when starting.
One of the obvious example is the alertmanager url: If you omit http for example, it will start up and only fail the first time a alert is triggered.

### PR DESCRIPTION
**Pull request by grobie**
Fixes #1455.

Why is URL validation so hard in golang ...

@discordianfish @fabxc

### CODE REVIEW

**Comment by fabxc** [comment]
:+1: nice

**Comment by grobie** [comment]
I assumed we want to be strict and not nice about the the scheme.

**Comment by discordianfish** [comment]
Yay :rocket: :) That was quick. LGTM!

File: cmd/prometheus/config.go [hunk 1]
```diff
@@ -301,6 +303,23 @@ func parseInfluxdbURL() error {
 	return nil
 }
 
+func validateAlertmanagerURL() error {
+	if cfg.notifier.AlertmanagerURL == "" {
+		return nil
+	}
+	if ok := govalidator.IsURL(cfg.notifier.AlertmanagerURL); !ok {
+		return fmt.Errorf("Invalid Alertmanager URL: %s", cfg.notifier.AlertmanagerURL)
```
Comment (juliusv) [inline cmd/prometheus/config.go]: lower-case error messages

Comment (fabxc) [inline cmd/prometheus/config.go]: @grobie ping

**Pull request merged**